app/javascript/__tests__/data/RailsListingPricingTableUnits/listing-pricing-table-units-default.ts

```diff
@@ -0,0 +1,200 @@
+import type { RailsListingPricingTableUnit } from "../../../api/types/rails/listings/RailsListingPricingTableUnit"
+
+export const pricingTableUnits: RailsListingPricingTableUnit[] = [
```

Comment:
Is this data used anywhere? I feel like this file is a work-in-progress, and was forgotten about during the handoff.

Reply:
It doesn't look like it is being used anywhere, should be deleted IMO